web/dispers/dispers.go

```diff
@@ -80,7 +80,25 @@ func selectAddresses(c echo.Context) error {
 		ListOfAddresses: finallist,
     })
 }
-  
+
+/*  
+*
+*
+Target'S ROUTES : those functions are used on route ./dispers/target/
+*
+*
+*/
+func getData(c echo.Context) error {
+
+	var localQuery dispers.InputT
```

Comment:
localQuery is quite confusing as variable name. The local query is *one* parameter among others